src/project-2.js

```diff
@@ -11,15 +17,31 @@ const greeting = (language) => {
   // language: 'English' -> 'Hello!'
   // language: 'Spanish' -> 'Hola!'
   // if language is undefined return 'Hello!'
+  if (language === 'German') {
+    return 'Guten Tag!';
+  } else if (language === 'English') {
+    return 'Hello!';
+  } else if (language === 'Spanish') {
+    return 'Hola!';
+  }
+  return 'Hello!';
 };
 
 const isTenOrFive = (num) => {
   // return true if num is 10 or 5
   // otherwise return false
+  if (num === 10 || num === 5) {
+    return true;
+  }
+  return false;
 };
 
 const isInRange = (num) => {
   // return true if num is less than 50 and greater than 20
+  if (num < 50 && num > 20) {
+    return true;
+  }
+  return false;
 };
 
 const isInteger = (num) => {
```

Comment:
Here I'd suggest besides putting all the code on one line, I'd also suggest returning `num === Math.floor(num)` over modulo.